### PR TITLE
OpenSearch support

### PR DESCRIPTION
**Pull request by Aschen**
### How should this be manually tested?

  - Step 1 :
  - Step 2 :
  - Step 3 :

### CODE REVIEW

File: lib/service/storage/elasticsearch.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const _ = require('lodash');
 const { Client: StorageClient } = require('@elastic/elasticsearch');
 const ms = require('ms');
 const Bluebird = require('bluebird');
-const semver = require('semver');
+// const semver = require('semver');
```
Comment (xbill82) [inline lib/service/storage/elasticsearch.js:29]: NIT - Is this left on purpose?

File: lib/service/storage/elasticsearch.js [hunk 2]
```diff
@@ -170,9 +170,9 @@ class ElasticSearch extends Service {
 
     const { body: { version } } = await this._client.info();
 
-    if (version && !semver.satisfies(semver.coerce(version.number), '>= 7.0.0')) {
-      throw kerror.get('version_mismatch', version.number);
-    }
+    // if (version && !semver.satisfies(semver.coerce(version.number), '>= 7.0.0')) {
```
Comment (xbill82) [inline lib/service/storage/elasticsearch.js:173]: 👳‍♂️ same-same?

**Review by xbill82 — APPROVED** [review]

**Review by alexandrebouthinon — COMMENTED** [review]
What about a new settings into the kuzzlerc like `ignoreStorageEngineVersionCheck`?

**Comment by Aschen** [comment]
This is still a draft, I just opened this PR to run the CI

**Comment by xbill82** [comment]
oh sorry

**Comment by Aschen** [comment]
Closing this PR. We need a proper SDK support

**Pull request closed without merging**